icons/cpu.svg

```diff
@@ -9,14 +9,14 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect width="16" height="16" x="4" y="4" rx="2" ry="2" />
-  <rect width="6" height="6" x="9" y="9" />
-  <line x1="9" x2="9" y1="2" y2="4" />
-  <line x1="15" x2="15" y1="2" y2="4" />
-  <line x1="9" x2="9" y1="21" y2="22" />
-  <line x1="15" x2="15" y1="20" y2="22" />
-  <line x1="20" x2="22" y1="9" y2="9" />
-  <line x1="20" x2="22" y1="14" y2="14" />
-  <line x1="2" x2="4" y1="9" y2="9" />
-  <line x1="2" x2="4" y1="14" y2="14" />
+  <rect x="4" y="4" width="16" height="16" rx="2" />
+  <rect x="9" y="9" width="6" height="6" />
+  <path d="M15 2v2" />
+  <path d="M15 20v2" />
+  <path d="M2 14h2" />
+  <path d="M2 9h2" />
+  <path d="M20 14h2" />
+  <path d="M20 9h2" />
+  <path d="M9 2v2" />
+  <path d="M9 20v2" />
```

Comment:
```suggestion
  <path d="M15 2v2" />
  <path d="M15 20v2" />
  <path d="M2 15h2" />
  <path d="M2 9h2" />
  <path d="M20 15h2" />
  <path d="M20 9h2" />
  <path d="M9 2v2" />
  <path d="M9 20v2" />
```

I think these two lines are probably also misaligned and not on purpose, so maybe fix these?